### PR TITLE
View

### PR DESCRIPTION
**Pull request by Brenda309**
**Project requirements**
Implement the design from the sneak peek wireframes.
Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/posts/_post.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          <div class="post_likes_counter">Likes: <%= post.likes_counter %></div>
+        </div>
+      </div>
+    <% end %>
+    <ul class="post_comments">
+    <% if post.comments.empty? %>
+      <li>No comments</li>
+    <% else %>
+      <% post.recent_5_comments.each do |comment| %>
+        <%= render comment %>
```
Comment (Davidosky007) [inline app/views/posts/_post.html.erb]: * Kindly make sure your app handles all routes as shown in the wireframe given, as after adding users using the rails console i still can't create a post and cant view either. ensure you are able to test your views at your end 
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Brenda309 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR description 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/_post.html.erb (hunk 1 above)
Comment (Brenda309) [reply app/views/posts/_post.html.erb]: Hello @Davidosky007, Thanks for the review but I am sorry, I am a little bit confused.   I clearly see the issues you are mentioning but on my side, everything is working as it should. I can create posts even when all routes are implemented and all working well. could you help me to investigate further more why this is happening? That way I can be awere of which part to change

Comment (Davidosky007) [reply app/views/posts/_post.html.erb]: Are you free for a call?

Comment (Brenda309) [reply app/views/posts/_post.html.erb]: Yes, I am

Comment (Davidosky007) [reply app/views/posts/_post.html.erb]: As you can see from the screenshot I shared I couldn't create a post

Comment (Brenda309) [reply app/views/posts/_post.html.erb]: Yes, I am

Comment (Brenda309) [reply app/views/posts/_post.html.erb]: my zoom user name => " Brenda Wihogora"

Comment (Davidosky007) [reply app/views/posts/_post.html.erb]: My slack handle is @davidosky

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Brenda309,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/_post.html.erb (hunk 1 above)
Comment (Davidosky007) [reply app/views/posts/_post.html.erb]: @future CR all works fine now 😃 ensure you populate the database with data 👍

**Pull request merged**